
https://gitlab.com/alfredchen/linux-prjc/-/issues/80

Date:   Wed Apr 26 13:37:09 2023 +0800

    sched/alt: [Sync] cd7f55359c90 sched: add sched_numa_find_nth_cpu()

diff --git a/kernel/sched/topology.c b/kernel/sched/topology.c
index c69f7c8a8f7f..21256b848f0b 100644
--- a/kernel/sched/topology.c
+++ b/kernel/sched/topology.c
@@ -2755,5 +2755,10 @@ int sched_numa_find_closest(const struct cpumask *cpus, int cpu)
 {
 	return best_mask_cpu(cpu, cpus);
 }
+
+int sched_numa_find_nth_cpu(const struct cpumask *cpus, int cpu, int node)
+{
+	return cpumask_nth(cpu, cpus);
+}
 #endif /* CONFIG_NUMA */
 #endif
